src/channels/infrastructure/persistence/entities/channel.entity.ts

```diff
@@ -0,0 +1,43 @@
+import { EntityRelationalHelper } from 'src/utils/relational-entity-helper';
+import {
+  Column,
+  CreateDateColumn,
+  DeleteDateColumn,
+  Entity,
+  ManyToOne,
+  PrimaryGeneratedColumn,
+  UpdateDateColumn,
+} from 'typeorm';
+import { Channel } from '../../../domain/channel';
+import { ChannelTypeEntity } from '../../../../channel-types/infrastructure/persistence/relational/entities/channel-type.entity';
+@Entity({
+  name: 'channel',
+})
+export class ChannelEntity extends EntityRelationalHelper implements Channel {
+  @PrimaryGeneratedColumn()
+  id: number;
+
+  @Column({ type: Number })
+  owner: number;
+
```

Comment:
In channel/owner we have to implement many-to-one relationship and the owner must be of type `UserEntity`